website/content/docs/configuration/adaptive-overload-protection.mdx

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+page_title: Adaptive Overload Protection - Configuration
+description: |-
+  Vault Enterprise provides Adaptive Overload Protection to automatically
+  prevent workloads from overloading different resources of the Vault servers.
+---
+
+# `adaptive_overload_protection`
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/beta.mdx'
+
+The `adaptive_overload_protection` stanza allows operators to control overload
+protection features. Changes to this stanza require a server restart to take
+effect.
```

Comment:
```suggestion
Configure the `adaptive_overload_protection` stanza to control the overload
protection features for your Vault server.

<Warning title="Restart required">

You must restart your Vault server for the stanza updates to take effect.

</Warning>
```

Reply:
We should probably make the restart warning a partial so we can include in elsewhere.